src/Miru/Queuing/Jobs.cs

```diff
@@ -6,15 +7,30 @@ namespace Miru.Queuing
     public class Jobs
     {
         private readonly IBackgroundJobClient _backgroundJobClient;
+        private readonly IRecurringJobManager _recurringJobManager;
 
-        public Jobs(IBackgroundJobClient backgroundJobClient)
+        public Jobs(IBackgroundJobClient backgroundJobClient, IRecurringJobManager recurringJobManager)
         {
             _backgroundJobClient = backgroundJobClient;
+            _recurringJobManager = recurringJobManager;
         }
 
         public void PerformLater<TJob>(TJob job) where TJob : IJob
         {
             _backgroundJobClient.Enqueue<JobFor<TJob>>(m => m.Execute(job, CancellationToken.None));
         }
+
+        public string PerformLater<TJob>(TJob job, string cronExpression) where TJob : IJob
+        {
+            var jobId = Guid.NewGuid().ToString();
+            _recurringJobManager.AddOrUpdate<JobFor<TJob>>(jobId,m => m.Execute(job, CancellationToken.None),
+                cronExpression, TimeZoneInfo.Local);
```

Comment:
When a call to a method is long, would be nice to split one parameter per line:

```csharp
Method(
   p1,
   p2,
   p3);

Reply:
I agree with you! For sure, that way will be easier to read and understand.